src/main/java/eu/mihosoft/vrl/v3d/CSG.java

```diff
@@ -100,11 +100,248 @@
     private static OptType defaultOptType = OptType.NONE;
     private OptType optType = null;
     private PropertyStorage storage;
+	private Bounds bounds;
 
     private CSG() {
         storage = new PropertyStorage();
     }
-
+	/**

```

Comment:
these are all helper functions that end up calling existing API calls. I added these here because doing so significantly cleaned up the user code, increasing readability of user scripts. Additions like this are specifically for users that use scripts in flat editors (Like JavaSCAD, BowlerStudio) where the IDE will not generate syntax for them.